numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -4277,19 +4255,11 @@ setup_scalartypes(PyObject *NPY_UNUSED(dict))
     SINGLE_INHERIT(Byte, SignedInteger);
     SINGLE_INHERIT(Short, SignedInteger);
 
-#if NPY_SIZEOF_INT == NPY_SIZEOF_LONG
-    INHERIT_INT(Int, SignedInteger);
-#else
     SINGLE_INHERIT(Int, SignedInteger);
-#endif
 
```

Comment:
```suggestion
```

Reply:
???

Reply:
Ah, remove blank lines

Reply:
opst, that was supposed to remove the blank line

Reply:
wait it should, now that its just one line, I think its good to group the ints OTOH, should also remove the one above then.

Reply:
I think github is just rendering this confusingly, I think the commit suggestion button will do the right thing